vendor/github.com/rcrowley/go-metrics/LICENSE

```diff
@@ -0,0 +1,29 @@
+Copyright 2012 Richard Crowley. All rights reserved.
```

Comment:
did you mean to add this file?

Reply:
IMHO, third party libraries should be vendorised "AS-IS"